src/State/SearchResult/actions.ts

```diff
@@ -1,41 +1,26 @@
 import * as types from './types';
 
-export const searchResultPageLoad = params => ({
+export const searchResultPageLoad = (params: any) => ({
   type: types.SEARCH_RESULT_PAGE_LOAD,
   payload: params
 });
 
-export const searchHandleInput = e => ({
+export const searchHandleInput = (e: { target: { value: any } }) => ({
```

Comment:
I believe there's a React.FormEvent or some type like that we can use

Reply:
Yeah thats good one. Added React.event